CHANGELOG.rst

```diff
@@ -12,6 +12,7 @@ Minor Changes
 -------------
 
 - firewall_rules - icmpv6 type - add support for vyos sw >= 1.4.
+- vyos_facts - support 1.3 show version output
```

Comment:
```suggestion
```
Please make this a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments) instead of modifying the changelog directly.